app/instance-initializers/axe-component.js

```diff
@@ -54,6 +55,15 @@ export function initialize() {
   let turnAuditOff = configuredTurnAuditOff || false;
 
   Component.reopen({
+    /**
+     * An Ember service which resolves concurrent axe.run v3 issue.
+     *
+     * @public
+     * @type {Object}
+     * @see(https://github.com/dequelabs/axe-core/issues/1041)
+     */
+    concurrentAxe: service(),
```

Comment:
Can you tweak this to:

```js
concurrentAxe: service('concurrent-axe'),
```

I know that with the default resolver the argument to `inject.service()` is optional here, but there are a few alternate resolvers out there that are much more strict (and would not normalize `concurrent-axe` into `concurrentAxe`).

Reply:
Of course, should be fixed now.